src-plugins/itkFilters/itkFiltersToolBox.cpp

```diff
@@ -107,7 +116,7 @@ itkFiltersToolBox::itkFiltersToolBox ( QWidget *parent ) : medAbstractSelectable
     d->addFilterWidget = new QWidget(this);
     d->addFilterValue = new QDoubleSpinBox;
     d->addFilterValue->setMaximum ( 1000000000 );
-    d->addFilterValue->setValue ( 100.0 );
+    d->addFilterValue->setValue ( itkFiltersAddProcess::defaultAddValue );
```

Comment:
I don't want to be a pain in the neck but ("prétérition") when I see this, I can't help but wonder if it wasn't better when the default parameter was set in the toolbox and not in the process ?

Reply:
(pffiou, i had to do some research about the "prétérition" word. Well done XD)
I don't mind putting the default parameters in the toolboxes or processes in fact.

Reply:
I really disagree here. It is always encouraged to place hard-coded values in constants or macros.

Reply:
Ok but can't we put "constants or macros" in the toolbox ?

Reply:
(we discussed this "live earlier with @LoicCadour) I understand that some default values don't make sense for the processes. For example, you would never use the add process directly without explicitly giving the value to add. On the other hand, there are cases where a default value is logical, like kernel shape for example. The idea was just to have the same default values for the toolbox and the processes, and so put them in one place.

Reply:
But we could also separate the two and have default values for the toolbox and default values for the processes. There is no reason to remove the default value constants we already coded for the processes, we could simply add separate ones in the toolbox class and consider they are not related. The initial reason why I wanted them in constants is:
1. It is good practice
2. The initial argument was that the toolbox and the process should have the same default value.

Reply:
(if you want to move that, it could be done in an other PR. We need to merge this PR almost "now")

Reply:
As long as we change that in a *coming-very-soon* PR, ok then